config/configtls/configtls.go

```diff
@@ -236,7 +312,7 @@ func (c TLSServerSetting) LoadTLSConfig() (*tls.Config, error) {
 		return nil, fmt.Errorf("failed to load TLS config: %w", err)
 	}
 	if c.ClientCAFile != "" {
-		certPool, err := c.loadCert(c.ClientCAFile)
+		certPool, err := c.loadCertFile(c.ClientCAFile)
```

Comment:
(Just for a comment) I think in the future we'd probably want to support inline client certificate chains for authentication, but it's not something we need right now so I think it's OK to descope it.